ruby/classes_and_objects/lib/account.rb

```diff
@@ -0,0 +1,27 @@
+class Account
+  attr_accessor :name, :account_no, :balance
```

Comment:
Do we need to create these accessors?

Reply:
yes sir, we need accessors. if we don't use accessors then we cann't set and get the value of instance variable. that's we need it. accessor is a implemented method in ruby which wright getter and setter for a variable.